Issue #3619: NeedBraces need clear description of allowSingleLineStatement (#14133)
checkstyle/checkstyle: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/blocks/needbraces/Example3.txt

   ++count;
while (value.incrementValue() < 5);  // violation, empty loop body not allowed
switch (num) {

>   case 1: counter++; break;          // OK, 'break' is not counted to allow compact view

Reviewer: ```suggestion
  case 1: counter++; break;        // OK, 'LITERAL_CASE' is not in configured tokens
```
Same as above.
